Quickstart.md

```diff
@@ -0,0 +1,197 @@
+WORKDIR /app
+
+# install compose if using it as the orchestrator
+RUN pip install docker-compose
+
+COPY requirements.txt .
+RUN pip install --requirement requirements.txt
+COPY src .
+
+ARG APP_VERSION=latest
+ENV APP_VERSION=${APP_VERSION}
+```
+
+We can see in the dockerfile, it is referring to a requirements.txt file which we will need to create. Go ahead and create a requirements.txt file in the same location as your Dockerfile and enter in the following `bsl-appcli==1.3.4` 
+This will install the appcli package in the image.
+
+### Create a docker compose file
+You will need to create a docker compose file which will be placed in the baseline folder. Give it the default name docker-compose.yml
+
+We will be using an Echo-Server for our example application which is useful for testing purposes as it replicates the request sent by the client and sends it back. More information on this container can be found [here](https://ealenn.github.io/Echo-Server/pages/quick-start/docker.html#run)
+
+Paste in the following code into your docker compose file.
+
+```Dockerfile
+version: '3'
+services:
+  echo-server:
+    image: ealen/echo-server:0.5.1
+```
+
+
+### Directory check
+
+Before we build our application please make sure your directory reflects the following structure
+
+```
+├── src
+│   ├── resources
+│   │   ├── settings.yml
+│   │   ├── stack-settings.yml
+│   │   ├── templates
+|   |           ├── baseline
+|   |           |       ├── docker-compose.yml
+|   |           ├── configurable
+|   | 
+│   ├── myapp.py
+|
+├── requirements.txt
+├── Dockerfile    
+```
+
+
+### Build the container
+
+```
+docker build -t brightsparklabs/myapp --build-arg APP_VERSION=latest .
+```
+
+### View and Run the installed script
+
+While it is not mandatory to view the script before running, it is highly recommended.
+
+```
+docker run --rm brightsparklabs/myapp:latest install
+```
+
+Once you can successfully see the script printed on the command line, you are ready to install
+
+```
+docker run --rm brightsparklabs/myapp:latest install | sudo bash` 
+```
+
+### Intialise
+
+Once you've succesfully ran the install script you can initialise the app by typing in the following command
+
+```
+/opt/brightsparklabs/myapp/production/myapp configure init`
+```
+
+### Applying the settings
+
+Before starting the services we need to run the configure apply command which will apply  the settings from the configuration.
+
+```
+/opt/brightsparklabs/myapp/production/myapp configure apply
+```
+
+### Running the service
+
+if you type in service, it will list all the commands you can use to control the services. The main ones are service start and service stop
+
+```
+/opt/brightsparklabs/myapp/production/myapp service start
+```
+
+You can check to if your service is still running by typing in the following docker command
+
+```
+docker ps
+```
+
+More commands can be found in the appcli readme [here](https://github.com/brightsparklabs/appcli)
```

Comment:
s/readme/README

Uppercase `README` is a general convention in software engineering.